examples/algo-example/src/app/algo.asa-transfer.example.ts

```diff
@@ -0,0 +1,81 @@
+import { TatumAlgoSDK, TransferAlgoBlockchain } from '@tatumio/algo'
+import { Currency, TransactionHash } from '@tatumio/api-client'
+import { isTestnet, sdkArguments } from '../index'
+import { BigNumber } from 'bignumber.js'
+
+/**
+ * Note: not able to burn ASA if creator doesnt own all assets
+ * https://developer.algorand.org/docs/get-details/asa/#destroying-an-asset
+ */
+
+export async function algoAsaTransferExample() {
+  const algoSDK = TatumAlgoSDK(sdkArguments)
+
+  // generate "from" and "to" addresses for wallets
+  // https://apidoc.tatum.io/tag/Algorand#operation/AlgorandGenerateWallet
+  const { address, secret } = algoSDK.wallet.generateWallet()
+  const fromPrivateKey = secret
+  const recipientAddress = algoSDK.wallet.generateWallet()
+  const to = recipientAddress.address
+
+  // FUND YOUR ACCOUNT WITH ALGOs FROM https://bank.testnet.algorand.network/
+
+  // Send Algos to a recipient account using private key, to be able accept token
+  // https://apidoc.tatum.io/tag/Algorand#operation/AlgorandBlockchainTransfer
+  const txData = (await algoSDK.transaction.signedTransaction(
```

Comment:
This algoSDK.transaction.signedTransaction is meant to be like this? I though it should be encapsulated like this:
algoSDK.transaction.send.signedTransaction